packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -437,6 +437,8 @@ module.exports = ({
        */
       {
         test: /\.js$/,
+        include: /node_modules/,
+        exclude: sourceFolders,
```

Comment:
maybe we should include this in our docs. what would happen if a user passed "node_modules" as part of their sourceFolders 🤔

Reply:
hm, maybe we can just document it. or provide a way to override this behaviour. maybe "optimizeVendorJavascript" can be a setting?

Reply:
Sure. I can do that tomorrow.